pokete_data/maps.py

```diff
@@ -379,6 +379,13 @@
         "extra_actions": None,
         "poke_args": None
     },
+    "playmap_39": {
+        "height": 69,
+        "width": 144,
+        "pretty_name": "Agrawos",
+        "extra_actions": None,
+        "poke_args": None
+    },
```

Comment:
Doesn't there have to be a definition of the map in `pokete_data/map_data.py`?

Reply:
Yes, I planned to do that. This is a map that will be added to the game later.